rust/otap-dataflow/crates/otap/src/syslog_cef_receiver.rs

```diff
@@ -349,6 +415,34 @@ impl local::Receiver<OtapPdata> for SyslogCefReceiver {
     }
 }
 
+/// RFC-aligned metrics for Syslog CEF receiver.
+///
+/// Key fields follow the component universal telemetry RFC for receivers:
+/// received.items.<signal>.<outcome>
+#[metric_set(name = "syslog_cef.receiver.metrics")]
+#[derive(Debug, Default, Clone)]
+pub struct SyslogCefReceiverMetrics {
+    /// Number of log records successfully received and forwarded
+    #[metric(unit = "{item}")]
+    pub received_items_logs_success: Counter<u64>,
+
+    /// Number of log records that failed to be received (parse errors, etc.)
+    #[metric(unit = "{item}")]
+    pub received_items_logs_failure: Counter<u64>,
+
+    /// Number of log records refused by downstream (backpressure/unavailable)
+    #[metric(unit = "{item}")]
+    pub received_items_logs_refused: Counter<u64>,
+
+    /// Number of TCP connections accepted
+    #[metric(unit = "{conn}")]
+    pub tcp_connections_accepted: Counter<u64>,
```

Comment:
Just knowing the number of accepted connections isn't super helpful. I think a better metric would be to track the number of active connections. We could use an `UpDownCounter<u64>` to track it.